templates/views/job.html

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/default.html" %}
+
+{% block content %}
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-10 col-sm-offset-1 col-md-8 col-md-offset-2">
+				<article>
+					<p>
+						<a href="/jobs">&larr; back to jobs</a>
+					</p>
+					<hr>
+					{% if not data.job %}
+						<h2>Job not found</h2>
+					{% else %}
+						<header>
+							<h1 class="post-title">{{ data.job.title }}</h1>
+							<h5 class="text-muted">Added
+								{% if data.job.publishedDate %}
+									on {{ data.job._.publishedDate.format("MMMM Do, YYYY") }}
+								{% endif %}
+							</h5>
+							<h5 class="text-muted">
+								{% if data.job.categories and data.job.categories.length %}
+									in {% for cat in data.job.categories %}<a href="/blog/{{ cat.key }}">{{ cat.name }}</a>{% if loop.index < data.job.categories.length %}, {% endif %}{% endfor %}
```

Comment:
@maryam-olabisi Please check the link here. Seems like you forgot to make a change i.e. href="/blog/ instead of href="/jobs/

Reply:
On the existing blog page, I noticed there was a category list that had jobs on it. I assumed this was to be a general categorization. Should I put that for jobs separately?

![aaaaaaa](https://user-images.githubusercontent.com/31415689/67085040-0606ad00-f196-11e9-8b37-708cf3ab1c39.png)

Reply:
Yes @maryam-olabisi, the category here should be for the jobs and should link to each job category. It is independent of/different from the Jobs category on the blog